modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]DefineAPISchemasWithinAnAPIGroup/AllowUsersToEditSchemaProperties.testcase

```diff
@@ -0,0 +1,311 @@
+			APIBuilderUI.goToEditRelatedEntryInEditApplication(
+				entity = "Schemas",
+				name = "University",
+				tabEntity = "Properties",
+				title = "My-app");
+
+			Click(
+				key_text = "name",
+				locator1 = "APIBuilder#PROPERTY_CONTAINER");
+		}
+
+		task ("Then 'name' field is placed in Properties") {
+			AssertTextEquals(
+				key_commentValue = "name",
+				locator1 = "PublicationsChanges#COMMENT_VALUE",
+				value1 = "name");
+		}
+	}
+
+	@priority = 4
+	test CanSearchThroughPropertiesFromTheList {
+		property portal.acceptance = "true";
+
+		task ("And Given 'University' Schema based on University object created") {
+			SchemaAPI.createAPISchema(
+				apiApplicationERC = "My-app",
+				mainObjectDefinitionERC = "university",
+				name = "University");
+		}
+
+		task ("And Given 'name', 'External Reference Code', 'Create Date', 'ID' fields placed in Properties") {
+			APIBuilderUI.goToEditRelatedEntryInEditApplication(
+				entity = "Schemas",
+				name = "University",
+				tabEntity = "Properties",
+				title = "My-app");
+
+			var fieldNameList = "name,External Reference Code,Create Date,ID";
+
+			for (var fieldName : list ${fieldNameList}) {
+				Click(
+					key_text = ${fieldName},
+					locator1 = "APIBuilder#PROPERTY_CONTAINER");
+			}
+		}
+
+		task ("When searching property 'External Reference Code' with Search box") {
+			Type(
+				locator1 = "TextInput#SEARCH",
+				value1 = "External Reference Code");
+		}
+
+		task ("Then only 'External Reference Code' present in Properties list") {
+			AssertElementNotPresent(
+				key_commentValue = "name",
+				locator1 = "PublicationsChanges#COMMENT_VALUE",
+				value1 = "name");
+
+			AssertElementNotPresent(
+				key_commentValue = "Create Date",
+				locator1 = "PublicationsChanges#COMMENT_VALUE",
+				value1 = "Create Date");
+
+			AssertElementNotPresent(
+				key_commentValue = "ID",
+				locator1 = "PublicationsChanges#COMMENT_VALUE",
+				value1 = "ID");
+
+			AssertTextEquals(
+				key_commentValue = "External Reference Code",
+				locator1 = "PublicationsChanges#COMMENT_VALUE",
+				value1 = "External Reference Code");
+		}
+	}
+
+	@priority = 4
+	test DroppedIntoPropertiesListRelatedObjectFieldIsDisabledInPropertiesSidebar {
+		property portal.acceptance = "true";
+
+		task ("And Given 'University' Schema based on University object created") {
+			SchemaAPI.createAPISchema(
+				apiApplicationERC = "My-app",
+				mainObjectDefinitionERC = "university",
+				name = "University");
+		}
+
+		task ("And Given fields of Address object definition present through View Related Objects'") {
+			APIBuilderUI.goToEditRelatedEntryInEditApplication(
+				entity = "Schemas",
+				name = "University",
+				tabEntity = "Properties",
+				title = "My-app");
+
+			APIBuilderUI.gotoRelatedObjectsProperties(relatedObjectName = "Postal Address");
+		}
+
+		task ("When dragging 'External Reference Code' of Address to 'Drop properties' box") {
```

Comment:
@magjed4289 Right now we only support click on a field to add a property, drag and drop to create a property is not supported. Should we update all "drag" into "Add" or "Create" in all task(steps) and case name? cc @Hijky

Reply:
@magjed4289 I have modified the 'drag' to 'add' and the case name 'CanDragAndDropPropertiesFromSidebarToList' to 'CanAddPropertiesFromSidebarToList'@cc @yanan